docs/UserGuide.md

```diff
@@ -167,13 +167,28 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/ROOM_NUMBER] [a/ADDRESS] [en
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* * Once `ADDRESS`, `ROOM_NUMBER`, `EMERGENCY_NAME`, `EMERGENCY_PHONE`, and/or `GRADUATION_YEAR` has been specified, you cannot remove these fields.
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 * You can remove all the person’s tags by typing `t/` without
   specifying any tags after it.
 
+[!Note]
```

Comment:
For all the [!Note] and [!Warning]s, this is apparently only supported by markdown and not by pdf / html so we decided to change it to these:
For notes: > \<span style="color:Gray"> NOTE! \</span> \<br>\
For tips: > \<span style="color:MediumSeaGreen"> TIP! \</span> \<br>\
For warnings: > \<span style="color:Tomato"> WARNING! \</span> \<br>

I think it's abit ugly and not very obvious though... do you think there is any better way to do it? Should we try to add images for tips notes and warnings like an exclamation mark, lightbulb and warning sign